Tests/TuistDependenciesTests/SwiftPackageManager/Utils/SwiftPackageManagerGraphGeneratorTests.swift

```diff
@@ -0,0 +1,76 @@
+import TuistDependencies
+import TuistGraph
+import XCTest
+@testable import TuistDependenciesTesting
+@testable import TuistSupportTesting
+
+class SwiftPackageManagerGraphGeneratorTests: TuistTestCase {
+    private var swiftPackageManagerController: MockSwiftPackageManagerController!
+    private var subject: SwiftPackageManagerGraphGenerator!
+
+    override func setUp() {
+        super.setUp()
+
+        swiftPackageManagerController = MockSwiftPackageManagerController()
+        subject = SwiftPackageManagerGraphGenerator(swiftPackageManagerController: swiftPackageManagerController)
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
should be called as last

Reply:
Fixed, but JFYI we have the same problem in many other test cases :D

Reply:
I know that, it's hard to control it everywhere. Maybe a custom swiftlint rule could help us 😅